app/src/js/utils/table-config/reconciliation-reports.js

```diff
@@ -85,24 +85,13 @@ const handleCsvDownloadClick = (e, reportName, dispatch) => {
 const handleDeleteClick = (e, value, dispatch, query) => {
   e.preventDefault();
   dispatch(deleteReconciliationReport(value)).then((response) => {
-    refreshReports(dispatch, value, query);
+    setTimeout(
```

Comment:
You're probably going to be irritated with me, but I meant that you should have a small delay in your other solution, I like that it retries (would be nice if it retried a finite number of times....).